packages/middleware/src/createServer.ts

```diff
@@ -33,6 +37,15 @@ async function createServer<
   config: MiddlewareConfig<TIntegrationContext>,
   options: CreateServerOptions = {}
 ): Promise<Server> {
+  const loggerManager = new LoggerManager<LoggerFactoryConfig>(
+    config,
+    (loggerConfig) => LoggerFactory.create(LoggerType.ConsolaGcp, loggerConfig)
+  );
+  const rawGlobalLogger = loggerManager.getGlobal();
+  const globalLogger = wrapLogger(rawGlobalLogger, () => ({
```

Comment:
loggerWithMetadata? enhancedLogger?
globalLogger is bad for 2 reasons:
1.you already have a global logger just with dumb name
2.it does not indicate the ehnancement/metadata